sageintacctsdk/apis/gl_detail.py

```diff
@@ -9,4 +9,4 @@
 class GLDetail(ApiBase):
     """Class for GL Detail APIs."""
     def __init__(self):
-        ApiBase.__init__(self, dimension='GLDETAIL')
+        ApiBase.__init__(self, dimension='GLDETAIL', post_legacy_method='create_sotransaction')
```

Comment:
@panasenco can you add the doc/reference link where you found `create_sotransaction`?

I am not able to find a reference to this in [GL Detail](https://developer.intacct.com/api/general-ledger/general-ledger-detail/) but found a reference for `create_sotransaction` in  [Order Entry Transactions](https://developer.intacct.com/api/order-entry/order-entry-transactions/)

Reply:
You're completely right, I'm sorry. It's order entry transactions we're dealing with. I'll update the PR.